docs/sphinx/install.rst

```diff
@@ -428,9 +425,145 @@ when connecting via SSH, which will launch the CUDA Quantum container and connec
+You should now see that job listed when you run the command
+
+.. code-block:: console
+
+    ngc base-command job list
+
+Once it has completed you can download the job results using the command
+
+.. code-block:: console
+
+    ngc base-command result download <job_id>
+
+replacing `<job_id>` with the id of the job you just submitted.
+You should see a new folder named `<job_id>` with the job log that contains 
+the output "Hello from DGX Cloud!".
+
+For more information about how to use the NGC CLI to interact with DGX Cloud, 
+we refer to the `NGC CLI documentation <https://docs.ngc.nvidia.com/cli/index.html>`__.
+
+Use JupyterLab
++++++++++++++++++++++++++++++++
+
+Once you can :ref:`run jobs on DGX Cloud <dgx-cloud-setup>`, you can launch an interactive job 
+to use CUDA Quantum with `JupyterLab <https://jupyterlab.readthedocs.io/en/latest/>`__ 
+running on DGX Cloud:
+
+.. code-block:: console
+
+    ngc base-command job run \
+      --name Job-interactive-001 --total-runtime 600s \
+      --image nvcr.io/nvidia/nightly/cuda-quantum:latest --result /results \
+      --ace <ace_name> --instance <instance_name> \
+      --port 8888 --commandline 'jupyter-lab-setup <my-custom-token> --port=8888'
+
+Replace `<my-custom-token>` in the command above with your own password that you can freely choose.
+You will use this password to authenticate with JupyterLab;
+Go to the `job portal <https://bc.ngc.nvidia.com/jobs>`__, click on the job you just launched, and click on the link
+under |:spellcheck-disable:|"URL/Hostname"|:spellcheck-enable:| in Service Mapped Ports. 
+
+.. note::
+
+  It may take a couple of minutes for DGX Cloud to launch and for the URL to become active;
+  if you encounter a "404: Not Found" error, be patient and try again in a couple of minutes.
+
+Once this URL opens, you should see the JupyterLab authentication page; enter the 
+password you selected above to get access to the running CUDA Quantum container.
+On the left you should see a folder with tutorials. Happy coding!
+
+Use VS Code
++++++++++++++++++++++++++++++++
+
+Once you can :ref:`run jobs on DGX Cloud <dgx-cloud-setup>`, you can launch an interactive job 
+to use CUDA Quantum with a local installation of `VS Code <https://code.visualstudio.com/>`_, 
+or the `VS Code Web UI <https://vscode.dev/>`__, running on DGX Cloud:
+
+.. code-block:: console
+
+    ngc base-command job run \
+      --name Job-interactive-001 --total-runtime 600s \
+      --image nvcr.io/nvidia/nightly/cuda-quantum:latest --result /results \
+      --ace <ace_name> --instance <instance_name> \
+      --commandline 'vscode-setup tunnel --name cuda-quantum-dgx --accept-server-license-terms'
+
+Go to the `job portal <https://bc.ngc.nvidia.com/jobs>`__, click on the job you just launched, and select the "Log"
+tab. Once the job is running, you should see instructions for how to log into the device.
+Follow the instructions to authenticate. Once you have authenticated, you can either 
```

Comment:
I'm probably just missing this, but I didn't see "how to log into the device" on that webpage. Through a bit of online searching, I eventually did `ngc base-command job exec --commandline 'bash' <jobid>`, and that allowed me to continue. I ran the installation for VSCode in the container.

I imagine it will be different if one runs with `--commandline 'vscode-setup tunnel --name cuda-quantum-dgx --accept-server-license-terms'`, but the user will still have to log in, so we might want to specify exactly how they do that instead of saying "you should see instructions for how to log into the device".

Reply:
Ah, the instructions for how to "log into the device" are coming from the vscode command, not from DGX Cloud, so yes, they only appear if you follow the job creation command that invokes vscode-setup tunnel. I can't give more specific instructions, because vscode will print a link along with a code to enter, and I don't know the code, so to know how to connect you *have* to look at the log.